platform/platform_windows.go

```diff
@@ -64,5 +128,41 @@ func GetArchInfo() (systemInfo map[string]interface{}, err error) {
 
 	systemInfo["kernel_name"] = "Windows"
 
+	// do additional work so that we don't panic() when the library's
+	// not there (like in a container)
+	si, sierr := netServerGetInfo()
+	if sierr != nil {
```

Comment:
```suggestion
	if sierr != nil {
	    systemInfo["family"] = "Unknown" //netServerGetInfo failed
	} else {
```
The code below assumes `si` has been initialized, so I think we should only run it if `netServerGetInfo` didn't fail?

Reply:
If it fails, the map entry will be empty, which is fine.

Reply:
My suggestion wasn't only adding a default `family` value to the map, it also flips the `if` condition which I think is not correct.

As it's written now, if it fails (ie: `sierr != nil`) then `si` will be null and reading `si.sv101_type` will then crash.

Reply:
oh, I see. You're right.